compiler/luci-interpreter/src/kernels/AveragePool2D.test.cpp

```diff
@@ -112,6 +112,38 @@ TEST(AveragePool2DTest, Uint8_1)
   EXPECT_THAT(extractTensorShape(output_tensor), ::testing::ElementsAreArray({1, 1, 2, 1}));
 }
 
+TEST(AveragePool2DTest, SInt16)
+{
+  Shape input_shape{1, 3, 5, 1};
+  std::vector<int32_t> ref_output_shape{1, 2, 2, 1};
+  std::vector<float> input_data{
+      -4, -3, -2, -1, 0,  //
+      1,  2,  3,  4,  5,  //
+      6,  7,  8,  9,  10, //
+  };
+  std::vector<float> ref_output_data{
+      0, 1.5, //
+      4.5, 6, //
+  };
+  Tensor input_tensor = makeInputTensor<DataType::S16>(input_shape, 0.5, 0, input_data);
```

Comment:
`makeInputTensor` quantizes a tensor using `quantize` function in `TestUtils.h`. The `quantize` function quantizes a tensor using the full range of a given data type (i.e., from `std::numeric_limits<T>::lowest()` to `std::numeric_limits<T>::max()`), which works well for uint8 quantization.

However, int16 quantization uses a symmetric range (from `std::numeric_limits<T>::lowest() +1` to `std::numeric_limits<T>::max()`). So we should extend `quantize` function in `TestUtils.h` to support symmetric quantization.

Reply:
Good catch, thanks!

Reply:
Fixed